src/main/resources/assets/packingtape/recipes/packingtape.json

```diff
@@ -5,8 +5,17 @@
     "item": "packingtape:tape"
   },
   "ingredients": [
-    { "item": "minecraft:slime_ball" },
-    { "item": "minecraft:string" },
-    { "item":"minecraft:paper" }
+    {
+      "type": "forge:ore_dict",
+      "ore": "slimeball"
+    },
+        {
```

Comment:
Your indentation is a bit weird. Can we get those opening braces aligned properly?